runtime/doc/map.txt

```diff
@@ -1431,6 +1431,43 @@ Possible values are (second column is the short name used in listing):
     -addr=other		  ?	other kind of range
 
 
+Incremental preview ~
+                                                  *:command-preview*
+It's possible to allow commands to be incrementally previewed when
+'inccommand' is enabled by using the -preview command attribute. Possible use
+cases are:
+	-preview		Always preview
+	-preview={func}		Preview only if {func} returns |TRUE|
+
+The {func} part must be a function with the following signature: >
+
+	:function {func}(CmdLine)
+
+The function is only given one argument, which is the entire command line. The
```

Comment:
I think I'd prefer a stronger tone here. Perhaps something like "The preview function accepts only one argument, which is the entire command line. Using two or more arguments is an error."? I think setting strong "Don't Do This" boundaries is useful :).

Also, does this mean that `-preview` with an argument needs to be used together with `-nargs=1`?

Reply:
No, `-preview` just takes the name of a function that's separate from the command replacement, and the function it takes can only take one argument. Also, I chose the wording based on the docs for the `-completion` attribute to keep consistency